.github/workflows/release-build.yml

```
@@ -117,7 +117,7 @@ jobs:
           source ~/pypi-check-env/bin/activate
 
           echo "Installing skypilot==${LATEST_PYPI_VERSION} from PyPI using uv..."
-          uv pip install --prerelease=allow "azure-cli>=2.65.0" "omegaconf>=2.4.0dev3"
+          uv pip install --prerelease=allow "azure-cli>=2.65.0"
```

`azure-cli` is on v2.71.0, so I'm guessing that whatever reason there was for allowing prereleases is now outdated and can be removed.

```suggestion
          uv pip install "azure-cli>=2.65.0"
```

---

@zpoint, I see that you recently added the `azure-cli` dependency for this workflow, but I don't see it being used for anything. Would it make sense to delete the whole line?